lib/ansible/modules/cloud/amazon/aws_secret.py

```diff
@@ -0,0 +1,358 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2018, REY Remi
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = r'''
+---
+module: aws_secret
+short_description: Manage secrets stored in AWS Secrets Manager.
+description:
+    - Create, update, and delete secrets stored in AWS Secrets Manager.
+author: "REY Remi (@rrey)"
```

Comment:
Do you mind making this a list and adding @slapula (`Aaron Smith <ajsmith10381@gmail.com>`) as an additional author to this module and the plugin since this is based on his work? If he resurfaces he'll be able to get pinged on things to help provide `shipit`s and maintain the module and plugin.

Reply:
No problem,  I changed it initially because I modified most of the code from the module.